ethereum/api/src/test/resources/prague_all_milestones_zero.json

```diff
@@ -0,0 +1,131 @@
+{
+  "config": {
+    "chainId":20211,
+    "homesteadBlock":0,
+    "eip150Block":0,
+    "eip155Block":0,
+    "eip158Block":0,
+    "byzantiumBlock":0,
+    "constantinopleBlock":0,
+    "petersburgBlock":0,
+    "istanbulBlock":0,
+    "muirGlacierBlock":0,
+    "berlinBlock":0,
+    "londonBlock":0,
+    "terminalTotalDifficulty":0,
+    "cancunTime":0,
+    "pragueTime":0,
+    "blobSchedule": {
+      "cancun": {
+        "target": 3,
+        "max": 6,
+        "baseFeeUpdateFraction": 3338477
+      },
+      "prague": {
+        "target": 6,
+        "max": 9,
+        "baseFeeUpdateFraction": 5007716
+      },
+      "osaka": {
+        "target": 9,
+        "max": 12,
+        "baseFeeUpdateFraction": 5007716
+      }
+    },
+    "clique": {
+      "period": 5,
+      "epoch": 30000
+    },
+    "depositContractAddress": "0x4242424242424242424242424242424242424242",
+    "withdrawalRequestContractAddress": "0x00A3ca265EBcb825B45F985A16CEFB49958cE017",
+    "consolidationRequestContractAddress": "0x00b42dbF2194e931E80326D950320f7d9Dbeac02"
+  },
+  "nonce":"0x42",
+  "timestamp":"0x0",
+  "extraData":"0x0000000000000000000000000000000000000000000000000000000000000000a94f5374fce5edbc8e2a8697c15331677e6ebf0b0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+  "gasLimit":"0x1C9C380",
+  "difficulty":"0x400000000",
+  "mixHash":"0x0000000000000000000000000000000000000000000000000000000000000000",
+  "coinbase":"0x0000000000000000000000000000000000000000",
+  "alloc":{
```

Comment:
Do we need all this extra config? Ideally we'd trim it down to the minimal valid config to use as a test fixture, for readability and repo size.

Reply:
I trimmed the genesis file down to the minimum.